### DISCUSSION TITLE
Global leaks anonymous reporting

### DISCUSSION THREAD

**Discussion by MakoEOS** [globaleaks/globaleaks-whistleblowing-software#3486] [Q&A] [answered]
Hi,
In the company where I work we are setting up a reporting page with Globaleaks and I was wondering if it would be possible to report anonymously without having to register as a user.

**Comment by evilaliv3** [comment, marked as answer]
Yes absolutely.

Whistleblowing does never require whistleblowers  to register on the site
to file a report.
